### PR TITLE
Updated ubuntu to 16.04

### PR DESCRIPTION
**Pull request by soumasish**
Bumped Ubuntu version to 16.04
https://jira.mesosphere.com/browse/DCOS-34279

### CODE REVIEW

File: universe/marathon.json.mustache [hunk 1]
```diff
@@ -35,8 +35,7 @@
 {{#service.spark-history-server-url}}
         "SPARK_HISTORY_SERVER_URL": "{{service.spark-history-server-url}}",
 {{/service.spark-history-server-url}}
-        "JAVA_HOME": "/usr/lib/jvm/jdk1.8.0_172/jre",
-        "LD_LIBRARY_PATH": "/opt/mesosphere/lib:/opt/mesosphere/libmesos-bundle/lib:/usr/lib",
+        "LD_LIBRARY_PATH": "/opt/mesosphere/libmesos-bundle/lib",
```
Comment (soumasish) [inline universe/marathon.json.mustache]: What's the rationale behind removing the JAVA_HOME from here?

File: docker/Dockerfile [hunk 2]
```diff
@@ -38,9 +38,9 @@ RUN apt-get update && \
 # lib mesos
 RUN mkdir -p /opt/mesosphere/ && \
     cd /opt/mesosphere && \
-    curl -L -O https://downloads.mesosphere.io/libmesos-bundle/libmesos-bundle-1.10-1.4.x-a35c801.tar.gz && \
-    tar zxf libmesos-bundle-1.10-1.4.x-a35c801.tar.gz && \
-    rm libmesos-bundle-1.10-1.4.x-a35c801.tar.gz && \
+    curl -L -O https://downloads.mesosphere.io/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz && \
```
[the remark this answers is not in the document]
Comment (akirillov) [reply docker/Dockerfile]: @elezar CI runs the latest version from this template https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise/testing/master/cloudformation/ee.single-master.cloudformation.json, it is supposed to be 1.13-dev afaik

[the remark this answers is not in the document]
Comment (akirillov) [reply docker/Dockerfile]: @elezar should we stick to libmesos-bundle 1.11 then to be aligned with SDK-based frameworks?

File: universe/marathon.json.mustache [hunk 3]
```diff
@@ -35,8 +35,6 @@
 {{#service.spark-history-server-url}}
         "SPARK_HISTORY_SERVER_URL": "{{service.spark-history-server-url}}",
 {{/service.spark-history-server-url}}
-        "JAVA_HOME": "/usr/lib/jvm/jdk1.8.0_172/jre",
```
[the remark this answers is not in the document]
Comment (akirillov) [reply universe/marathon.json.mustache]: @elezar it is set in Docker image

File: docker/Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (elezar) [reply docker/Dockerfile]: Let's check the nighties

**Pull request merged**